keps/sig-scheduling/5234-dra-resourceslice-mixins/kep.yaml

```diff
@@ -0,0 +1,51 @@
+title: DRA ResourceSlice Mixins
+kep-number: 5234
+authors:
+  - "@mortent"
+  - "@klueska"
+owning-sig: sig-scheduling
+participating-sigs:
+  - sig-node
+  - sig-autoscaling
+status: implementable
+creation-date: 2025-04-09
+reviewers:
+  - "@pohly"
+  - "@johnbelamaric"
+  - "@thockin"
+approvers:
+  - "@mrunalp" # SIG-Node
+  - "@alculquicondor" # SIG-Scheduling
+  - "@MaciekPytel" # SIG-Autoscaling
```

Comment:
Same here. @jackfrancis or @raywainman maybe?

Reply:
I would add @towca here who has been working the most on DRA stuff on the autoscaling side

Reply:
I'll give it a review pass now (but yes either @towca or myself would make sense here as Maciek has transitioned away from active SIG Autoscaling duty)

Reply:
Added @jackfrancis as he has already reviewed it.